rootex/core/renderer/post_processor.cpp

```diff
@@ -153,6 +153,55 @@ class MonochromePostProcess : public PostProcess
 	}
 };
 
+Variant CustomPostProcess::loadRTVAndSRV(const Event* event)
```

Comment:
shift all `CustomPostProcess::` to bottom of the file

Reply:
fixed